modules/gitops-creating-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-creating-rolloutmanager-custom-resource_{context}"]
+= Creating RolloutManager custom resource
+
+To manage progressive delivery of deployments by using the Argo Rollouts feature in {gitops-title}, you must create and configure a RolloutManager custom resource (CR) in the namespace of your choice. By default, any new `argo-rollouts` instance has permission to manage resources only in the namespace where it is deployed. But you can use Argo Rollouts in multiple namespaces as required.
```

Comment:
Please ensure that you are following the guidelines for object formatting throughout the doc. It should always be "`RolloutManager` custom resource (CR)" if you're using the literal name of an object. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Also, sentences should not start with "but" - see previous comment about restructuring the sentence, or use "However, "